features/multi-column.yml

```diff
@@ -0,0 +1,20 @@
+name: Multi-column layout
+description: Multi-column layout flows an element's content across one or more content areas in a single row, without affecting the `display` properties of its children.
```

Comment:
I feel like "content areas" should be replaced with "columns". That would make it easier to understand I think.

The "without affecting the display" last part feels a bit obscure to me. I would suggest either clarifying it or removing it.

Reply:
I used "content areas" to avoid a recursive description like "Multi-column layout flows content into columns", but maybe this is a gray area?

"Without affecting the `display` properties" is a distinction from `flexbox` and `grid` that is called out in both the spec and posts comparing these layout options. It would probably be clearer to state the "`display` property" (singular), as that the key point: this property of child elements is unaffected using multi-column on the parent

Reply:
Re "display property", I think I get it. Grid and Flex blockify their items, so inline items become blocks. But Multi-column doesn't do that. If that's the main thing to convey, then what about this: "without affecting the `display` property of its children. For example, inline elements remain inline, unlike in Grid or Flexbox layouts, where grid and flex items become block elements".
Might be too much details, and too long. But I thought giving a concrete example would help make this part of the description easier to understand (i.e. less spec-talk). Up to you.